frontend/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: ["eslint:recommended", "plugin:vue/vue3-recommended"],
```

Comment:
I have included the rules suggested by eslint in the project to avoid problems, we can check the recommended rules with a 'tick' next to them on the [rules page](https://eslint.org/docs/latest/rules/) and we can discuss possible problems